docs/user/README.md

```diff
@@ -29,6 +29,7 @@ Welcome to the Flight Control user documentation.
   * [Managing Applications](managing-devices.md#managing-applications)
   * [Using Device Lifecycle Hooks](managing-devices.md#using-device-lifecycle-hooks)
   * [Monitoring Device Resources](managing-devices.md#monitoring-device-resources)
+  * [Non-Blocking Image Prefetch Management](non-blocking-prefetch.md) - Guide to understanding and troubleshooting background image downloads
```

Comment:
we have been adding smaller feature docs to managing-devices.md just wondering if this qualifies for seperate doc file. just something to consider.